server/api/index.js

```diff
@@ -1,7 +1,7 @@
 const router = require('express').Router();
 module.exports = router;
 
-router.use('/users', require('./users'));
+router.use('/users', require('./users').router);
```

Comment:
what is the .router for?

Reply:
Not entirely sure. Just know that when Lisa and I were trying to fix the routes, this seemed to fix it. But not sure honestly.